go.mod

```diff
@@ -2,4 +2,4 @@ module github.com/aquasecurity/tfsec-github-commenter-action
 
 go 1.15
 
-require github.com/owenrumney/go-github-pr-commenter v0.0.15
+require github.com/owenrumney/go-github-pr-commenter v0.0.16
```

Comment:
tfsec check aws-ecr-enforce-immutable-repository failed. 

Description: Repository tags are mutable.

Severity: HIGH

For more information, see:

- https://aquasecurity.github.io/tfsec/v1.19.1/checks/aws/ecr/enforce-immutable-repository/
- https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ecr_repository